### PR TITLE
✨ Add Mapbox

### LINKED ISSUE #20: Ass a user I want to choose my campsite on a Map
- [] install MapBox
- [] Map all campsites to Mapbox
- [] Add button, where the user can add a campsite to a special day

### PR DESCRIPTION
**Pull request by m-wiemers**
Fix #20 

- add API function `getCity`
- add API function `getCampsites`

### CODE REVIEW

File: pages/map.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import React, { useEffect, useState } from "react";
+import ReactMapGL, { Marker } from "react-map-gl";
+import AussiTentIcon from "../components/icons/AussiTentIcon";
```
Comment (mewiemers) [inline pages/map.tsx]: see the warning s here, is AussiTentIcon necessary?

**Review by mewiemers — APPROVED** [review]
super Cowboy 🤠

File: pages/map.tsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+  useEffect(() => {
+    getCampSites().then(setCampsites);
+  }, []);
+
+  if (!campsites) {
+    return <div>Loading...</div>;
+  }
+
+  const campsite = campsites.map((campsite) => (
```
Comment (phlgr) [inline pages/map.tsx]:
```suggestion
  const campsiteMarkers = campsites.map((campsite) => (
```

File: pages/map.tsx [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+  return (
+    <div className={styles.map}>
+      <ReactMapGL
+        {...viewport}
+        mapboxApiAccessToken={apiToken}
+        mapStyle={mapStyle}
+        onViewportChange={(viewport) => setViewport(viewport)}
+      >
+        <div>{campsite}</div>
```
Comment (phlgr) [inline pages/map.tsx]:
```suggestion
        <div>{campsiteMarkers}</div>
```

**Review by phlgr — COMMENTED** [review]
Gj 🚀

**Pull request merged**